routers/web/repo/editor.go

```diff
@@ -146,11 +146,6 @@ func editFile(ctx *context.Context, isNewFile bool) {
 		}
 
 		blob := entry.Blob()
-		if blob.Size() >= setting.UI.MaxDisplayFileSize {
-			ctx.NotFound(err)
-			return
-		}
-
 		dataRc, err := blob.DataAsync()
```

Comment:
We should not read the file if file is too large.

Reply:
7684e1720d980161ec61c7e126455689df50c3fa